bloom-instance/logging.tf

```diff
@@ -1,3 +1,24 @@
 resource "aws_s3_bucket" "logging_bucket" {
   bucket_prefix = "${var.name_prefix}-logging"
 }
+
+resource "aws_s3_bucket_policy" "log_bucket_policy" {
```

Comment:
are we going to create this resource if logging is not enabled? This appears to be a generic logging bucket  policy but below it's specifying an ALB target.

Reply:
There will also be a need to store CloudFront logs at the very least, so I wanted to avoid overly strict naming.  Since there can only be one bucket policy on any given bucket, I'll just add more statements in as necessary.  There will be multiple CloudFront distributions and I also suspect that there may be a need to add the flexibility to create more than one ALB, so I'll just replace the single value with a set of loops later on.

Reply:
I see, you'll be adding in more `Resource`s to this policy in the future. Thanks!